db/seeds.rb

```diff
@@ -116,3 +116,14 @@
     metadata.categories.find_or_create_by!(name: category_name)
   end
 end
+
+[
```

Comment:
as i've been coming across hard-coded data i'm trying to add them to the seeds file. since i was mucking around in Reports and Forms to complete this workshop_log crud, i found this list of Forms to autogenerate.